config/settings.yml

```diff
@@ -1743,3 +1743,7 @@ vye:
 schema_contract:
   appointments_index: 'modules/vaos/app/schemas/appointments_index.json'
   test_index: 'spec/fixtures/schema_contract/test_schema.json'
+
+ogc:
+  form21a_service_url:
+    url: https://ogccotst1.dva.va.gov:4701/api/v2/applications/Form21as
```

Comment:
Can you extract this (seemingly?) sensitive value into the manifest repo and parameter store?

Reply:
Hi Lindsey! So, this is an internal URL on the VA network. I see [other URLs similarly added to this file](https://github.com/department-of-veterans-affairs/vets-api/blob/c19873a0a518877d1e30bf04bf1944becd4a674a/config/settings.yml#L557-L558). Could you shed light on why we might think this is sensitive? Happy to remove this and make a follow-on if so!

Reply:
The convention that we followed while moving over to k8s was to place urls on the VA network into param store

Reply:
Sounds good! I removed it from this PR and made a follow-on issue.

See commit:
https://github.com/department-of-veterans-affairs/vets-api/pull/17120/commits/2490c12d2047e9533ecff41b2963015d6d8b813e

And ZH Issue:
https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/87177